.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -1,6 +1,5 @@
 name: Issue
 description: Report an issue with one of the samples in this repository. 
-labels: ["needs triage"]
```

Comment:
Was this change intentional, as well as the deletion of `SUPPORT.md`?

Reply:
Yes, that was intentional. We currently aren't using `SUPPORT.md` as a template, and @odoe had suggested removing the triage tag.